StitchCoreServicesMongoDbRemote/Sources/StitchCoreServicesMongoDbRemote/Collectible.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+import MongoSwift
+
+public protocol Collectible {
```

Comment:
[minor] This doesn't seem to be getting used anywhere.

Reply:
Yep, didn't mean to check this in